docs/0.1/getting-started/tutorial.md

```diff
@@ -1,7 +1,104 @@
 ---
 layout: docs
 title: Tutorial
-description: Coming Soon
+description: Getting Set-up in Microshare
 group: getting-started
 toc: true
 ---
+
+
+This tutorial will provide an simple review of Microshare. In this tutorial we will:
+
+1. Create an account  
+2. Get an access token  
+3. Post data to Microshare  
+4. Make a rule to govern this data  
+5. Create a robot to moniter this data  
+6. Make a dashboard to view this data  
+
+This process will require Postman. It is avaliable to download and install [here](https://www.getpostman.com/)
+
+## Post Data to Microshare
+
+Make a post call using the same parameters as before, but with two notable additions. 
+
+1. Include the data you wish to upload into the body of the call in JSON format. For example:
+
+    {"Test":"Data"}
+    
+2. Include a header with a key of "recType" 
+    * This will create a space for the data and attribute the above value as the recType's name  
+        * It is recommended that your first recType follow a simple naming convention I.E. "yourFirstName.yourLastName"
+
```

Comment:
A screenshot from postman here would have been best. Check the assets, there should be an image you can use.